packages/app/src/components-styled/content-header/index.tsx

```diff
@@ -28,6 +28,7 @@ const HeaderBox = styled.header<{
     mt: 0,
     ml: x.skipLinkAnchor ? '-100vw' : x.hasIcon ? undefined : 5,
     pl: x.skipLinkAnchor ? '100vw' : undefined,
+    pointerEvents: 'none',
```

Comment:
This results in an unresponsive UI: users will not be able anymore to interact with the text which is an accessibility drawback. Also things like anchors are not clickable anymore

Reply:
I'm not entirely sure I understand the current implementation, but perhaps its sufficient to replace this line with:
```suggestion
    pointerEvents: x.skipLinkAnchor ? 'none' : 'auto',
```

Reply:
I forgot to commit my last change! Please check again